frontend/catalyst/jit.py

```diff
@@ -590,15 +589,6 @@ def closure(qnode, *args, **kwargs):
 
         with Patcher(
             (qml.QNode, "__call__", closure),
-            # !!! TODO: fix jax.scipy numerical failures with properly fetched lapack calls
```

Comment:
That was a short-lived fix 😆

Reply:
No kidding! Short-lived TODOs are the best TODOs :muscle: